assets/js/script.js

```diff
@@ -37,7 +37,8 @@ function fn1(e) {
  
    fetch(newRecipe)
       .then(response => {
-         if (!response.ok){
+         if (response.status !== 200){
+            window.location.href = "./statusError.html";
             throw Error("ERROR")
```

Comment:
Great execution—thanks for taking care of this Luke! Two things from me:

1. Why only show the error page when the response status is 200? 
2. Do we need line 42, ie. `throw Error("ERROR")` or is this covered by showing them the error HTML page?

Reply:
Hi Hayley, 

So the !== 200 means that every status code out of 200 redirects to the statusError.html (atleast thats how it worked when i tested haha).

No we will not need the throw error that is just for testing purposes